internal/net/grpc/interceptor/server/metric/metric.go

```diff
@@ -54,51 +54,32 @@ func MetricInterceptors() (grpc.UnaryServerInterceptor, grpc.StreamServerInterce
 		return nil, nil, errors.Wrap(err, "failed to create completedRPCs metric")
 	}
 
+	record := func(ctx context.Context, method string, err error, latency float64) {
+		code := codes.OK // default error is success when error is nil
+		if err != nil {
+			st, _ := status.FromError(err)
+			if st != nil {
+				code = st.Code()
+			}
```

Comment:
do you think we need to handle the case that if the status cannot be parsed?
```suggestion
			if st != nil {
				code = st.Code()
			} else {
				code = codes.Unknown
			}
```

Reply:
status.FromError returns Unknown status if parse failure.
we don't need handle it.

Reply:
FYI: https://github.com/vdaas/vald/blob/main/internal/net/grpc/status/status.go#L170